flytekit/clis/flyte_cli/main.py

```diff
@@ -403,6 +403,12 @@ def _render_schedule_expr(lp):
     required=True,
     help="Whether or not to set schedule as active."
 )
+_workflow_state_choice = _click.option(
+    "--state",
+    type=_click.Choice(["active", "archived"]),
+    required=True,
+    help="Whether or not to set a workflow as actively visible in the UI."
```

Comment:
actively visible?

Reply:
trying to tie it back to 'active'... i'll reword

Reply:
We should say Ui/ cli. Default is active only